### PR TITLE
pybind11, new recipe

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: dev-libs/pybind11/pybind11-2.4.3.recipe [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+
+BUILD_PREREQUIRES="$BUILD_PREREQUIRES
```
Comment (korli) [inline dev-libs/pybind11/pybind11-2.4.3.recipe]: please remove $BUILD_PREREQUIRES

**Comment by korli** [comment]
otherwise looks good.

File: dev-libs/pybind11/pybind11-2.4.3.recipe (hunk 1 above)
Comment (Begasus) [reply dev-libs/pybind11/pybind11-2.4.3.recipe]: Still working on it, should also be "any" as there is no compiler involved? (at least it's working that   way, wip) :)

**Comment by Begasus** [comment]
This is a wip recipe to get support for scipy multi python version

File: dev-libs/pybind11/pybind11-2.4.3.recipe (hunk 1 above)
Comment (Begasus) [reply dev-libs/pybind11/pybind11-2.4.3.recipe]: Updated ...

**Comment by korli** [comment]
> This is a wip recipe to get support for scipy multi python version

are you sure?

**Comment by Begasus** [comment]
I was looking into the missing dependencies for noteshrink and saw that scipy_python was missing the headers for this one at one point

**Comment by Begasus** [comment]
atm I'm looking into scipy_python 2.4.1 and you can find pylibs11 in their requirements now: https://github.com/scipy/scipy/blob/345eb722978868788236a0f6ee12ccd0bd5c1553/INSTALL.rst.txt#prerequisites

**Comment by korli** [comment]
right, I would stick to scipy_python3-1.3.1.recipe, as that's what uses ArchLinux ATM. I think they dropped python2 support also, so you need two recipes.

**Comment by Begasus** [comment]
> right, I would stick to scipy_python3-1.3.1.recipe, as that's what uses ArchLinux ATM. I think they dropped python2 support also, so you need two recipes.

OK thanks, I'll look into 1.3.1 then, wanted to see if I could get both python versions (3.6 and 3.7) in one recipe, but getting some errors, worth investigating :) (or maybe create a task for it?)

**Comment by Begasus** [comment]
Looks OK from my side

**Pull request merged**